docs/src/test-api/class-testinfo.md

```diff
@@ -2,7 +2,7 @@
 * since: v1.10
 * langs: js
 
-`TestInfo` contains information about currently running test. It is available to any test function, [`method: Test.beforeEach`] and [`method: Test.afterEach`] hooks and test-scoped fixtures. `TestInfo` provides utilities to control test execution: attach files, update test timeout, determine which test is currently running and whether it was retried, etc.
+`TestInfo` contains information about currently running test. It is available to any test function, before/after hook or a test-scoped fixture. `TestInfo` provides utilities to control test execution: attach files, update test timeout, determine which test is currently running and whether it was retried, etc.
```

Comment:
The links are kind of nice for navigating between related hooks/methods etc :bowtie: 

Also at the risk of sounding knit picky - best to be explicit where possible (before/after hooks could be interpreted as beforeAll and afterAll only for people coming from Mocha land). Consider:

```
`TestInfo` contains information about the currently running test. It is available to all test functions, all test hook functions ([`method: Test.beforeAll`], [`method: Test.beforeEach`], [`method: Test.afterEach`], and [`method: Test.afterAll`]) and inside test-scoped fixtures.

`TestInfo` provides utilities to control test execution: attach files, update test timeout, determine which test is currently running and whether it was retried, etc.
```